### PR TITLE
Added options for searching anime by season on kitsu

### PR DESCRIPTION
**Pull request by dymattic**
### Added options for searching anime by season on kitsu

I wanted to have the option to display and select anime by year and season, so the following was implemented:

1.  AnimeInfo Class ( handles the kitsu stuff )
2.  Option to search for anime by season in download_cli method
3.  Same for seasonal_cli
4. Changed some logic in download_cli so downloads start in bulk after episode selection for multiple shows is done

### CODE REVIEW

File: anipy_cli/anime_info.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        :param season_year: Year of Season
+        :type season_year: int
+        :param season_name: name of the season (spring,summer,fall,winter)
+        :type season_name: str
+        :return: list of Anime
+        :rtype: list
+        """
+
+        assert (
+            season_name in self.season_names
```
Comment (sdaqo) [inline anipy_cli/anime_info.py:33]: we could use the .lower()  method here to prevent the only lower case

**Comment by sdaqo** [comment]
Okay, this is pretty cool though two things before merging, could you add the extra dependencies to setup.py, and add a option in the config to deactivate the prompt, some users may not want to always type no if they don't want to search the seasons. Other than that, I tested it and it works.

**Comment by sdaqo** [comment]
lgtm, thanks for the pr!

**Comment by dymattic** [comment]
Lol that was quick haha ^^ 
You're welcome, I find this tool quite handy :3 @sdaqo

**Comment by dymattic** [comment]
Also the kitsu api library needs at least python 3.9 if I am not mistaken. Should we add that to readme?

**Comment by sdaqo** [comment]
I added it to the install procedure so it will enforce it automatically

**Comment by sdaqo** [comment]
Hey @dymattic could you deploy your kitsu.py project to pypi, since pypi doesnt allow deps from other sources, and so I wont be able to upload it to pypi.

**Comment by sdaqo** [comment]
Oh nevermind, you already did that I will include that version in this script, that okay for you?

**Comment by dymattic** [comment]
Oh yea regarding that, I submitted a PR to [MrArkon/kitsu.py](https://github.com/MrArkon/kitsu.py), but it hasn't been approved and merged yet.
The changes I made are necessary for us though.


Alternatively I could make some changes to the AnimeInfo class, so it works with the version of kitsu.py that is currently available on pypi.

**Pull request merged**